packages/flatpak.rb

```diff
@@ -43,12 +43,16 @@ def self.patch
     Digest::SHA256.hexdigest(File.read(patch_filename)) == patch_sha256
     puts 'patch downloaded'.lightgreen
     system 'patch  -p 1 --forward < patch || true'
+    # Source has libglnx repo as submodule
+    FileUtils.rm_rf('libglnx')
+    system 'git clone --depth=1 https://gitlab.gnome.org/GNOME/libglnx.git libglnx '
```

Comment:
Should we be checking out a tagged release?

Reply:
No tags for 3 years... but I just have it checking out a specific commit now.